jumpscale/sals/zos/gateway.py

```diff
@@ -20,6 +22,17 @@ class GatewayGenerator:
 
     def __init__(self, explorer):
         self._gateways = explorer.gateway
+        
+    def correct_domain(self, domain):
```

Comment:
can you add a docstring explaining what this method does please. regex are always hard to read and deduct what they do.

Reply:
done